src/code/code_800EC960.c

```diff
@@ -2740,58 +2752,59 @@ s8 func_800F3990(f32 arg0, u16 arg1) {
     return ret | 1;
 }
 
-void func_800F3A08(u8 bankIdx, u8 entryIdx, u8 channelIdx) {
+void Audio_SetSoundProps(u8 bankIdx, u8 entryIdx, u8 channelIdx) {
```

Comment:
does `Props` stand for "properties"? if yes, since it's also an actual word (as in, accessories) I think "Settings" would be better even if slightly longer

(or "Info", "Data"? not exactly sure what does this function do, apply sound info of `entryIdx` in `bankIdx` to `channelIdx`?)

Reply:
if properties is a better fitting word, i dont think its an issue to just spell out the whole thing personally

Reply:
Yep, it means properties: reverb, volume, pitch, pan, etc. Settings seems like a worse name. I changed props -> properties.

> not exactly sure what does this function do, apply sound info of entryIdx in bankIdx to channelIdx?

it computes sounds properties loosely based on the `entryIdx` sound request, and then applies those properties to a sequence channel. (Sequence players/channels/layers is what produces sound, by playing notes with various properties; here we communicate with that subsystem.)